lib/pwwka/transmitter.rb

```diff
@@ -1,5 +1,6 @@
 begin  # optional dependency
   require 'resque'
+  require 'resque-retry'
```

Comment:
This was needed because `enqueue_at` doesn't exist in resque…only in resque-retry